winpr/include/winpr/wtypes.h

```diff
@@ -49,17 +57,39 @@
 #define NEAR
 #endif
 
+#if defined(HAVE_STDINT_H)
```

Comment:
@akallabeth but the code does #define - not typedef so I think @hardening is right ... there is something wrong here